azkaban-common/src/test/java/azkaban/user/XmlUserManagerTest.java

```diff
@@ -108,13 +114,10 @@ public void testAutoReload() throws Exception {
 
     // Modify the password for user8
     // TODO : djaiswal : Find a better way to modify XML
-    List<String> lines = new ArrayList<>();
-    List<String> origLines = new ArrayList<>();
-    Path filePath = Paths.get(path);
-    FileTime origModifiedTime = Files.getLastModifiedTime(filePath);
+    final FileTime origModifiedTime = Files.getLastModifiedTime(filePath);
     System.out.println("File modification time = " + origModifiedTime.toString());
```

Comment:
There's a lot of use of System.out.println -- please remove or convert to log.info/debug

Reply:
Thanks. I just updated them to use log.